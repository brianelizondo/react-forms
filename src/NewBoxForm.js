import React, { useState } from "react";

function NewBoxForm(){
    return (
        <div className="NewBoxForm">
            
        </div>
    );
}

export default NewBoxForm;