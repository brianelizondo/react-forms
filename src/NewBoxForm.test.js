import React from "react";
import { render } from "@testing-library/react";
import NewBoxForm from "./NewBoxForm";

// Smoke Tests
it("'NewBoxForm' component renders without crashing", function() {
    render(<NewBoxForm />);
});

// Snapshot Tests
it("'NewBoxForm' matches snapshot", function() {
    const {asFragment} = render(<NewBoxForm />);
    expect(asFragment()).toMatchSnapshot();
});
