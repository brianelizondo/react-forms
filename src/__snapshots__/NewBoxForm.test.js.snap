// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`'NewBoxForm' matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="NewBoxForm"
  >
    <form>
      <label
        for="width"
      >
        Box width:
      </label>
      <input
        id="width"
        name="width"
        placeholder="box width"
        value=""
      />
      <label
        for="height"
      >
        Box height:
      </label>
      <input
        id="height"
        name="height"
        placeholder="box height"
        value=""
      />
      <label
        for="color"
      >
        Box color:
      </label>
      <input
        id="color"
        name="color"
        placeholder="box color"
        value=""
      />
      <button>
        Add Box!
      </button>
    </form>
  </div>
</DocumentFragment>
`;
